src/resources/public_keys/public_keys.qrc

```diff
@@ -0,0 +1,6 @@
+<RCC>
+    <qresource prefix="/addons_signature">
+        <file>staging.der</file>
+        <file>production.der</file>
```

Comment:
Do we need a followup task to work with releng to ensure this key is updated before the next release where this feature goes live?

Reply:
Yes. I had a meeting with them to start the process.